addons/mail/static/src/components/channel_member_list_category/channel_member_list_category.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.ChannelMemberListCategory" owl="1">
         <h6 class="m-2" t-esc="title"/>
         <t t-foreach="members" t-as="member" t-key="member.localId">
-            <ChannelMember channel="channelMemberListView.channel" member="member"/>
+            <ChannelMember channel="(record.channelMemberListViewOwnerAsOnline and record.channelMemberListViewOwnerAsOnline.channel) or (record.channelMemberListViewOwnerAsOffline and record.channelMemberListViewOwnerAsOffline.channel)" member="member"/>
```

Comment:
Could we compute this, instead of this long conditional expression ?

Reply:
Changes done (introduced `ChannelMemberListCategoryView/channel`)